File: Epay3Client/src/types/SmtpConfig.ts
export interface SmtpConfig {
  smtpAddress: string;
  smtpPassword: string;
  smtpPort: string;
  smtpUseUser: boolean;
  smtpUser: string;
  fromAddress: string;
  fromAddressName: string;
  mailBody: string;
  mailSubject: string;
  registrationRequestEmail: string;
  testEmail: string;
  applicationUrl: string;
  companyName: string;
}

File: Epay3Service/Models/SmtpConfig.cs
namespace Epay3Service.Models;

public class SmtpConfig
{
    public string SmtpAddress { set; get; }
    public bool SmtpUseUser { set; get; }
    public int SmtpPort { set; get; }
    public string SmtpUser { set; get; }
    public string SmtpPassword { set; get; }
    public string RegistrationRequestEmail { set; get; }
    public string FromAddress { get; set; }
    public string FromAddressName { get; set; }
    public string TestEmail { get; set; }

    public string MailSubject { get; set; }

    public string MailBody { get; set; }
}